### PR TITLE
Adding code example in Event Subscriber concept; renaming getBuilder() into builder to address issue #138

### PR DESCRIPTION
**Pull request by meizabel**
Changes summary:
- Renaming getBuilder() into builder() in step Step 3 snippet spine.io,
- Adding a note to Command Handler and Event Subscriber concepts.

### CODE REVIEW

File: docs/concepts/command-handler.md [hunk 1]
```diff
@@ -9,5 +9,19 @@ type: markdown
 <h2 class="top">Command Handler</h2> 
 
 Command Handler is an object which receives commands, modifies the state of the application, and generates events if the modification was successful. 
+Here is a code example which shows how a method which handles this kind of messages looks like.
```
Comment (alexander-yevsyukov) [inline docs/concepts/command-handler.md]: This code must go under `Task Subscriber`.

File: docs/concepts/event-subsriber.md [hunk 2]
```diff
@@ -8,4 +8,20 @@ type: markdown
 ---
 <h2 class="top">Event Subscriber</h2> 
 
-Event Subscriber is an object that is subscribed to receive events.
\ No newline at end of file
+Event Subscriber is an object that is subscribed to receive events.
```
Comment (alexander-yevsyukov) [inline docs/concepts/event-subsriber.md]: This is an example of command handling, not event subscribing.

File: docs/concepts/command-handler.md (hunk 1 above)
Comment (meizabel) [reply docs/concepts/command-handler.md]: Should we create a new term entry for the Concepts section and name it `Task Subscriber` then?

File: docs/concepts/event-subsriber.md (hunk 2 above)
Comment (meizabel) [reply docs/concepts/event-subsriber.md]: LGTM
I will place this example to the `Command Handler` section then.

File: docs/concepts/command-handler.md (hunk 1 above)
Comment (alexander-yevsyukov) [reply docs/concepts/command-handler.md]: No, I meant “Event Subscriber”.

File: docs/concepts/event-subsriber.md [hunk 3]
```diff
@@ -10,4 +10,21 @@ type: markdown
...
 
-You can consider [Projection](/concepts/projection.md) as a type of Event Subscriber. 
\ No newline at end of file
+Here is a code example which shows how a method which handles this kind of messages looks like.
+
+  ```
+  final class TaskProjection
+      extends Projection&lt;TaskId, TaskItem, TaskItemVBuilder&gt; {
+      ...
+      @Subscribe
```
Comment (alexander-yevsyukov) [inline docs/concepts/event-subsriber.md]: In the text we speak about _one_ method. The source code sample has two. Please leave only the one which handles the `TaskCompleted`.

**Comment by alexander-yevsyukov** [comment]
@meizabel, please also rename the PR into something meaningful.

File: docs/concepts/event-subsriber.md (hunk 3 above)
Comment (meizabel) [reply docs/concepts/event-subsriber.md]: LGTM

**Pull request merged**